components/calendar/src/calendar.rs

```diff
@@ -90,9 +89,13 @@ pub trait Calendar {
         smallest_unit: DateDurationUnit,
     ) -> DateDuration<Self>;
 
-    /// The [`AnyCalendarKind`] corresponding to this calendar,
-    /// if one exists. Implementors outside of `icu::calendar` should return `None`
-    fn any_calendar_kind(&self) -> Option<AnyCalendarKind>;
+    #[cfg(feature = "ixdtf")]
```

Comment:
Issue: Not a fan of making trait functions gated on features.

This is a public trait, not a scaffold trait. The docs say it is "unstable" but not private/internal.

Reply:
ungated